jobs/service-fabrik-interoperator/templates/config/kubeconfig.yaml.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  CA = link('service-fabrik-apiserver').p('tls.apiserver.ca')
+  CA_BASE64 = Base64.strict_encode64(CA)
+%>
+---
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: "<%= CA_BASE64 %>"
+    server: "https://127.0.0.1:<%= link('service-fabrik-apiserver').p('port') %>"
```

Comment:
Is it  better to fetch ip of 'service-fabrik-apiserver' from job property and consume it, instead of hard-coding localhost's ip? If these two are not co-deployed in future by any chance, then required. Right? Just a thought.